addons/website_slides/views/slide_channel_invite_views.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record model="ir.ui.view" id="slide_channel_invite_view_form">
```

Comment:
id first, then model.

Reply:
The world is now a little better thanks to you <3